src/FieldWarning/Assets/Units/MobilityType.cs

```diff
@@ -115,4 +114,22 @@ private float GetSlopeFactor(Terrain terrain, Vector3 location, Vector3 directio
         float speed = 1.0f / (1.0f + overallSlopeFactor + directionalSlopeFactor);
         return Mathf.Max(speed - 0.1f, 0f);
     }
+
+    public static int GetIndexForConfig(MobilityConfig config)
+    {
+        foreach (MobilityType m in MobilityTypes) {
+            if (
+                m.SlopeSensitivity == config.SlopeSensitivity
+                && m.DirectionalSlopeSensitivity == config.DirectionalSlopeSensitivity
+                && m.PlainSpeed == config.PlainSpeed
+                && m.ForestSpeed == config.ForestSpeed
+                && m.WaterSpeed == config.WaterSpeed
+            ) {
```

Comment:
I just noticed this is inconsistent with the double indent/parens on the same line format for breaking lines